includes/sanitizers/class-amp-gtag-script-sanitizer.php

```diff
@@ -55,5 +55,32 @@ public function sanitize() {
 				ValidationExemption::mark_node_as_px_verified( $script );
 			}
 		}
+
```

Comment:
Performance improvement: we can `return` here if the number of scripts above was zero. No need to do the attribute query if we already know gtag isn't on the page.

Reply:
I guess it'll not come here if the `$scripts` is empty 🤔 .